pyproject.toml

```diff
@@ -48,6 +48,7 @@ dependencies = [
     'zarr',
     'rich',
     'rich-pixels',
+    'ipython'
```

Comment:
Ah I see that it's actually used in the code for the wandb logger? Then rather in the [wandb] and [all] optional groups.

So that we can install it using "pip install "careamics[wandb]"